src/Microsoft.Azure.WebJobs.Extensions.Kafka/Output/KafkaProducerFactory.cs

```diff
@@ -74,6 +74,12 @@ private string CreateKeyForConfig(ProducerConfig producerConfig)
         private IProducer<byte[], byte[]> CreateBaseProducer(ProducerConfig producerConfig)
         {
             var builder = new ProducerBuilder<byte[], byte[]>(producerConfig);
+            ILogger logger = this.loggerProvider.CreateLogger("Kafka");
+            builder.SetLogHandler((_, m) =>
+            {
+                logger.LogInformation($"Libkafka: {m?.Message}");
```

Comment:
Same as before, we could map the log level to have more information about the severity

Reply:
Done